src/App.tsx

```diff
@@ -21,6 +40,8 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            defaultValue={perPage}
+            onChange={({ target }) => changePerPage(target)}
```

Comment:
let's pass whole event or value of this target
```suggestion
            onChange={({ target }) => changePerPage(target.value)}
```